### ISSUE TITLE
Impress Flatpak: constant crashes on load, Connection to wayland server lost

### ISSUE THREAD

**Issue by firefoxlover** [flathub/org.libreoffice.LibreOffice#229]
I edit an Impress presentation. 

After some time, inserting a photo or something bigger, it crashes. These crashes never recover themselves, it always goes completely blank and I cant recover anything.

I launched the Flatpak from terminal, but no messages except that there way no Wayland connection anymore, when the Program already crashed. I also saw no "-debug" or "-verbose" flag for running the program, so this is all.

Specified App: org.libreoffice.LibreOffice	7.5.3.2	stable	flathub

--- Software ---
OS: Fedora Linux 38.20230522.0 (Kinoite)
KDE Plasma: 5.27.5
KDE Frameworks: 5.106.0
Qt: 5.15.9
Kernel: 6.2.15-300.fc38.x86_64
Compositor: wayland

--- Hardware ---
CPU: AMD Ryzen 5 PRO 3500U w/ Radeon Vega Mobile Gfx
RAM: 13.5 GB
GPU: AMD Radeon Vega 8 Graphics
Video memory: 2048MB

**Comment by stbergmann** [comment]
If you can make available a presentation that works with your reproducer (please specify which exact image to Ctrl+X, if there are multiple), I can take a look whether I might be able to reproduce it.

**Comment by ahjolinna** [comment]
I also have this issue now with openSUSE microOS, doesn't seem to affect just Impress it affects all of libreoffice

for me when I open anything it crashes after few seconds of usage, doesn't matter what I use/do


----

my system spec:

```
             .;ldkO0000Okdl;.                
         .;d00xl:^''''''^:ok00d;.            OS: openSUSE MicroOS
       .d00l'                'o00d.          Kernel: x86_64 Linux 6.3.2-1-default 
     .d0K^'  Okxoc;:,.          ^O0d.        Uptime: 3h 21m
    .OVVAK0kOKKKKKKKKKKOxo:,      lKO.       Packages: Unknown
   ,0VVAKKKKKKKKKKKKK0P^,,,^dx:    ;00,      Shell: bash 5.2.15
  .OVVAKKKKKKKKKKKKKk'.oOPPb.'0k.   cKO.     Resolution: 3072x1728
  :KVAKKKKKKKKKKKKKK: kKx..dd lKd   'OK:     DE: KDE 5.106.0 / Plasma 5.27.5
  lKlKKKKKKKKKOx0KKKd ^0KKKO' kKKc   lKl     WM: KWin_wayland
  lKlKKKKKKKKKK;.;oOKx,..^..;kKKK0.  lKl     GTK Theme: Breeze [GTK2],  [GTK3]
  :KAlKKKKKKKKK0o;...^cdxxOK0O/^^'  .0K:     Icon Theme: Papirus-Dark
   kKAVKKKKKKKKKKKK0x;,,......,;od  lKP      Disk: 16T / 21T (75%)
   '0KAVKKKKKKKKKKKKKKKKKK00KKOo^  c00'      CPU: AMD Ryzen 7 5700G with Radeon Graphics @ 16x 4.3GHz
    'kKAVOxddxkOO00000Okxoc;''   .dKV'       GPU: NVIDIA GeForce RTX 3060 Ti [driver: v525.116.04]
      l0Ko.                    .c00l'        RAM: 5594MiB / 15779MiB
       'l0Kk:.              .;xK0l'         
          'lkK0xc;:,,,,:;odO0kl'            
              '^:ldxkkkkxdl:^'

**Comment by firefoxlover** [comment]
It was very reproducible.

I dont know if its true, but both reported OSses have KDE. It seemed to have gotten a bit more stable after disabling baloo

```
balooctl disable && balooctl purge
```

**Comment by stbergmann** [comment]
So I cannot reproduce a crash when in your Präsentation-Ozean.odp I randomly Ctrl+X the large image on slide 8 "Chlorophyll".  (I'm running the latest `org.libreoffice.LibreOffice/x86_64/stable` 7.5.3.2 on Fedora 37 GNOME/Wayland.)

I'm not sure how a file indexer like <https://community.kde.org/Baloo> could be relevant here, I'd rather assume that some clipboard service could cause issues when LibreOffice puts the image in the clipboard upon the Ctrl+X.

When you install from Flathub both `org.freedesktop.Sdk//22.08` and the `org.libreoffice.LibreOffice.Debug` extension, you should be able to run `flatpak run --devel org.libreoffice.LibreOffice --backtrace` and hope that it will print a backtrace when it crashes.  If that does not work well, alternatively you can run `flatpak run --devel --command=gdb org.libreoffice.LibreOffice /app/libreoffice/program/soffice.bin` and then at the `(gdb)` prompt enter `run`, and when it crashes enter `thread apply all backtrace` to get a backtrace.

**Comment by ahjolinna** [comment]
yeah, disabling baloo didn't help me at all. I tried to run in safe mode and that didn't work, when I tried to `"flatpak run --devel org.libreoffice.LibreOffice --backtrace" `debug I get that logging is not enabled (didn't find any doc on how to enable it flatpak, I assume?)


anyway, here is at least : [strace.log](https://github.com/flathub/org.libreoffice.LibreOffice/files/11556050/strace.log)

**Comment by firefoxlover** [comment]
Yes I get the same gbd error "No debugging symbols found"

Weird, I dont know the reason.

--- Software ---
OS: Fedora Linux 38.20230523.0 (Kinoite)
KDE Plasma: 5.27.5
KDE Frameworks: 5.106.0
Qt: 5.15.9
Kernel: 6.2.15-300.fc38.x86_64
Compositor: wayland

--- Hardware ---
CPU: AMD Ryzen 5 PRO 3500U w/ Radeon Vega Mobile Gfx
RAM: 13.5 GB
GPU: AMD Radeon Vega 8 Graphics
Video memory: 2048MB

**Comment by stbergmann** [comment]
> Yes I get the same gbd error "No debugging symbols found"

Even then, can you use one of the two approaches I outlined to get a (symbol-less) backtrace?

**Comment by firefoxlover** [comment]
Thanks a lot for the tutorial on debug logs.

I really tried to make it crash, but I could not reproduce it. I copied and pasted everything on that presentation, nothing happened.

only explanation I can think of would be too much RAM usage, but afaik that was ok back then.


Specified App: 
org.libreoffice.LibreOffice	7.5.3.2	stable	flathub

--- Software ---
OS: Fedora Linux 38.20230608.0 (Kinoite)
KDE Plasma: 5.27.5
KDE Frameworks: 5.106.0
Qt: 5.15.9
Kernel: 6.3.5-200.fc38.x86_64
Compositor: wayland

--- Hardware ---
CPU: AMD Ryzen 5 PRO 3500U w/ Radeon Vega Mobile Gfx
RAM: 13.5 GB
GPU: AMD Radeon Vega 8 Graphics
Video memory: 2048MB
